package.json

```diff
@@ -25,15 +25,15 @@
     "url": "git://github.com/strongloop/loopback-api-definition.git"
   },
   "dependencies": {
-    "js-yaml": "^3.4.3",
-    "loopback-swagger": "^3.0.0"
+    "js-yaml": "^3.8.1",
+    "loopback-swagger": "^4.0.0"
   },
   "devDependencies": {
-    "chai": "^1.9.1",
-    "eslint": "^2.8.0",
-    "eslint-config-loopback": "^2.0.0",
-    "loopback": "^1.7.4",
-    "loopback-datasource-juggler": "^1.3.10",
-    "mocha": "^1.18.2"
+    "chai": "^3.5.0",
+    "eslint": "^3.16.1",
+    "eslint-config-loopback": "^8.0.0",
+    "loopback": "^3.4.0",
+    "loopback-datasource-juggler": "^3.2.0",
```

Comment:
This is no longer needed in `loopback@3`, juggler comes as a regular loopback dependency now.